<launch>
	<node name="database_node"    pkg="bwi_scavenger" type="database_node"    output="screen"/>
	<node name="perception_node"  pkg="bwi_scavenger" type="perception_node"  output="screen"/>
  <node name="move_node"        pkg="bwi_scavenger" type="move_node"        output="screen"/>
  <node name="find_object_node" pkg="bwi_scavenger" type="find_object_node" output="screen"/>
  <node name="conclude_node"    pkg="bwi_scavenger" type="conclude_node"    output="screen"/>
</launch>
